<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranslatorClass</class>
 <widget class="QMainWindow" name="TranslatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translator</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenGit"/>
    <addaction name="actionRecent"/>
    <addaction name="actionReload"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionReleaseCur"/>
    <addaction name="actionReleaseAll"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionProjectFile"/>
   </widget>
   <widget class="QMenu" name="menu_Option">
    <property name="title">
     <string>&amp;Option</string>
    </property>
    <addaction name="actionSetting"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Option"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockProjFile">
   <property name="windowTitle">
    <string>Project File</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="ProjFileWidget" name="projFileWidget"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionRecent">
   <property name="text">
    <string>Recent Project</string>
   </property>
  </action>
  <action name="actionOpenGit">
   <property name="text">
    <string>Creckout &amp;Git Project</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Reload Project</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Close Project</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionReleaseCur">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Release Current</string>
   </property>
  </action>
  <action name="actionReleaseAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Release All</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Project Setting</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionProjectFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Project File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ProjFileWidget</class>
   <extends>QWidget</extends>
   <header>projfilewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../translator.qrc"/>
 </resources>
 <connections/>
</ui>
